NBitcoin.Tests/NodeBuilderEx.cs

```diff
@@ -1,59 +1,77 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
+﻿using System.Net;
 using System.Runtime.CompilerServices;
 
 namespace NBitcoin.Tests
 {
 	public class NodeBuilderEx
 	{
-		public static NodeBuilder Create([CallerMemberName] string caller = null)
+		public static NodeDownloadData GetNodeDownloadData()
 		{
-			//var builder = NodeBuilder.Create(NodeDownloadData.Litecoin.v0_17_1, Altcoins.Litecoin.Instance.Regtest, caller);
```

Comment:
please don't edit that

Reply:
The original method still exists below. It merely calls the two new methods that split out the network set and node data into two reusable methods (e.g. note that some of the altcoin tests need to know which network they are running against, for some reason)

Reply:
There is no point in doing that, now we need to modify 2 distinct places when 99% of the time we just want to uncomment 1 line.

Reply:
Can you suggest an improved way of determining which network set is in effect in the tests? It is necessary to inhibit certain tests from running when they are not running against the intended network (e.g. Liquid). If this is not done, at least one spurious test failure results across all other networks, which feels sloppy.